cuegui/cuegui/config/cuegui.yaml

```diff
@@ -1,5 +1,15 @@
 # Default CueGUI config file
 
+# If False, use the version number in the VERSION.in file
+# Otherwise, use the version cuegui.custom.cmd.version.beta (for beta) or cuegui.custom.cmd.version.stable (for stable)
+cuegui.use.custom.version: False
+# Used to show the CueGUI beta version, if the environment variable OPENCUE_BETA = 1 (setenv OPENCUE_BETA 1)
+# If the CueGUI is deployed using the SPK and SPAWN (https://getspk.io/), it is also possible to run the SPAWN command
+# to get the cuegui version or any other shell command.
```

Comment:
Rewrite this without mentioning SPK and SPAWN. They are open source but not publicly used here. Mentioning another project might just create more confusion.

Reply:
Done!

```python
# If the CueGUI beta version is enabled, it's possible to run a shell command to get the cuegui version or any other
# custom command specified in the environment configuration.```